lib/server/sthServer.js

```diff
@@ -49,135 +49,153 @@ var attendedRequests = 0;
+                sthLogger.warn(
+                    sthServerUtils.getContext(request),
+                    request.method.toUpperCase() + ' ' + request.url.path + ', event=' + JSON.stringify(event)
+                );
+            } else {
+                sthLogger.error(
+                    sthServerUtils.getContext(request),
+                    request.method.toUpperCase() + ' ' + request.url.path + ', event=' + JSON.stringify(event)
+                );
+            }
+        }
+    });
 
-  server.connection({
-    host: host,
-    port: port
-  });
+    server.connection({
+        host: host,
+        port: port
+    });
 
-  server.route([
-    {
-      method: 'GET',
-      path: '/STH/v1/contextEntities/type/{entityType}/id/{entityId}/attributes/{attrName}',
-      handler: sthGetDataHandler,
-      config: {
-        validate: {
-          headers: sthHeaderValidator,
-          query: {
-            lastN: joi.number().integer().greater(-1).optional(),
-            hLimit: joi.number().integer().greater(-1).optional(),
-            hOffset: joi.number().integer().greater(-1).optional(),
-            aggrMethod: joi.string().valid('max', 'min', 'sum', 'sum2', 'occur').optional(),
-            aggrPeriod: joi.string().required().valid('month', 'day', 'hour', 'minute', 'second').optional(),
-            dateFrom: joi.date().optional(),
-            dateTo: joi.date().optional(),
-            filetype: joi.string().optional(),
-            count: joi.boolean().optional()
-          }
-        }
-      }
-    },
-    {
-      method: 'GET',
-      path: '/version',
-      handler: sthGetVersionHandler
-    },
-    {
-      method: 'POST',
-      path: '/notify',
-      handler: sthNotificationHandler,
-      config: {
-        validate: {
-          headers: sthHeaderValidator
-        }
-      }
-    },
-    {
-      method: 'DELETE',
-      path: '/STH/v1/contextEntities',
-      handler: sthRemoveDataHandler,
-      config: {
-        validate: {
-          headers: sthHeaderValidator
-        }
-      }
-    },
-    {
-      method: 'DELETE',
-      path: '/STH/v1/contextEntities/type/{entityType}/id/{entityId}',
-      handler: sthRemoveDataHandler,
-      config: {
-        validate: {
-          headers: sthHeaderValidator
-        }
-      }
-    },
-    {
-      method: 'DELETE',
-      path: '/STH/v1/contextEntities/type/{entityType}/id/{entityId}/attributes/{attrName}',
-      handler: sthRemoveDataHandler,
-      config: {
-        validate: {
-          headers: sthHeaderValidator
+    server.route([
+        {
+            method: 'GET',
+            path: '/STH/v1/contextEntities/type/{entityType}/id/{entityId}/attributes/{attrName}',
+            handler: sthGetDataHandler,
+            config: {
+                validate: {
+                    headers: sthHeaderValidator,
+                    query: {
+                        lastN: joi
```

Comment:
I think that break this in 6 lines is excesive... It's much more clear:

```
lastN: joi.number().integer().greater(-1).optional(),
```

In general, this happens in many places in this PR. It would be great if you could have a general look and fix all the "excesive line breaking" cases to avoid them to happen. Thanks!

Reply:
Fixed  8b8777